Client/Frontend/Browser/BrowserViewController/BrowserViewController+Menu.swift

```diff
@@ -29,7 +30,72 @@ extension BrowserViewController {
         }
     }
     
-    func destinationMenuSection(_ menuController: MenuViewController) -> some View {
+    func privacyFeaturesMenuSection(_ menuController: MenuViewController) -> some View {
+        VStack(alignment: .leading, spacing: 0) {
+            Spacer(minLength: 15.0)
+            
+            Text(verbatim: Strings.PrivacyFeature.menuSectionTitle.uppercased())
```

Comment:
Don't need verbatim here btw, since you are passing a `String` and not a string literal

Reply:
Fix at https://github.com/brave/brave-ios/pull/4424/commits/67ce7646e8e57fc92c7941b2ddbdf30450aaadec